README.md

```diff
@@ -160,13 +160,14 @@ See unit test in xt folder for more examples
     youtubeAnalytics : v1,v1beta1 : http://developers.google.com/youtube/analytics/
     youtubereporting : v1 : https://developers.google.com/youtube/reporting/v1/reports/
 
-# AUTHOR
+# AUTHORS
 
-Pavel Serikov <pavelsr@cpan.org>
+- Pavel Serikov <pavelsr@cpan.org>
+- Peter Scott <peter@pscott.com.au>
 
 # COPYRIGHT AND LICENSE
 
-This software is copyright (c) 2017-2018 by Pavel Serikov.
+This software is copyright (c) 2017-2018 by Pavel Serikov, Peter Scott.
```

Comment:
copyright/authors/ and all documentation handled by dzil. no need to mess with it at all.

Reply:
the approach I've taken here is through the dist.ini which is what I assume dzil uses - you seem to keep correcting me on the consequences of these edits - am I not approaching this correctly? I shall look more closely at how you are handling it in your branches in case I'm missing something.